R/clipboard.R

```diff
@@ -7,9 +7,7 @@
 #'
 #' @note \code{read_clip} will not try to guess at how to parse copied text. If
 #'   you are copying tabular data, it is suggested that you use
-#'   \code{\link[readr]{read_delim}} or one of its sibling functions from the
-#'   readr package (\url{https://cran.r-project.org/package=readr}) to coax the
-#'   text into a data frame.
+#'   \code{xl2df}.
```

Comment:
Use \code{\link{function_name}} to create a hyperlink within the R documentation.